src/lang/en.json

```
@@ -0,0 +1,244 @@
+{
+  "loading": "Loading...",
+  "confirm": "Confirm",
+  "cancel": "Cancel",
+  "delete": "Supprimer",
+  "quit": "Quit",
+  "errror": "Error",
+  "welcome": {
+    "FirstInstallation": {
+      "message": "Welcome on Papillon!",
+      "subtitle": "The only app to manage all your school life in one place!",
+      "getStarted": "Let's start!",
+      "help": "Need help?",
+      "tos": "By continuing, you accept the {{termsOfService}} and the {{privacyPolicy}}.",
+      "terms": "terms of use",
+      "privacyPolicy": "privacy policy"
+    },
+    "ChangelogScreen": {
+      "title": "What's new?",
+      "pending": "Loading the news...",
+      "pendingSubtitle": "Getting the latest news from Papillon",
+      "error": "Unable to load the changelog",
+      "errorSubtitle": "Please check your internet connection, or try again later...",
+      "features": "New features",
+      "fixes": "Bug fixes"
+    }
+  },
+  "login": {
+    "ServiceSelector": {
+      "message": "To begin with, what's your school service?",
+      "highEdu": "Higher education",
+      "highEduSubtitle": "Universities, institutes, schools...",
+      "importV6": "Import my account"
+    },
+    "PronoteAuthSelector": {
+      "message": "What do you prefer to connect to Pronote?",
```

je pense mais pas sur 
```suggestion
      "message": "What do you prefer to use to connect to Pronote?",
```